sphinxcontrib/versioning/versions.py

```diff
@@ -243,6 +243,8 @@ def vpathto(self, other_version):
         return '{}.html'.format(__import__('posixpath').join(*components))
 
     def pathtopdf(self, other_version):
+        if self.pdf_file == None:
```

Comment:
```suggestion
        if self.pdf_file is None:
```
https://www.python.org/dev/peps/pep-0008/#programming-recommendations

> Comparisons to singletons like None should always be done with is or is not, never the equality operators.